### ISSUE TITLE
Elsa 2 - YesSQL - 2.0.0-rc2.102 failed startup initialization 

### ISSUE THREAD

**Issue by matt4446** [elsa-workflows/elsa-core#999]
I had expected the schema to change a bit based on a previous bug (https://github.com/elsa-workflows/elsa-core/commit/c7b24261510738238c18ae223da04d0b913a450b), but after letting it create the tables it still had the same error : 
```

System.ArgumentException: Expression of type 'System.Int32' cannot be used for parameter of type 'System.String' of method 'Boolean Contains[String](System.Collections.Generic.IEnumerable`1[System.String], System.String)' (Parameter 'arg1')
   at System.Dynamic.Utils.ExpressionUtils.ValidateOneArgument(MethodBase method, ExpressionType nodeKind, Expression arguments, ParameterInfo pi, String methodParamName, String argumentParamName, Int32 index)
   at System.Linq.Expressions.Expression.Call(MethodInfo method, Expression arg0, Expression arg1)
   at System.Linq.Expressions.MethodCallExpression2.Rewrite(Expression instance, IReadOnlyList`1 args)
   at System.Linq.Expressions.ExpressionVisitor.VisitMethodCall(MethodCallExpression node)
   at System.Linq.Expressions.MethodCallExpression.Accept(ExpressionVisitor visitor)
   at System.Linq.Expressions.ExpressionVisitor.Visit(Expression node)
   at Elsa.Persistence.YesSql.ExpressionHelper.TypeConversionVisitor`1.Visit(Expression node)
   at Elsa.Persistence.YesSql.ExpressionHelper.ConvertType[TFrom,TTo](Expression`1 from)
   at Elsa.Persistence.YesSql.ExpressionHelper.ConvertType[TFrom,TTo](Expression`1 from)
   at Elsa.Persistence.YesSql.Stores.YesSqlStore`2.AutoMapSpecification[TIndex](ISession session, ISpecification`1 specification)
   at Elsa.Persistence.YesSql.Stores.YesSqlWorkflowInstanceStore.MapSpecification(ISession session, ISpecification`1 specification)
   at Elsa.Persistence.YesSql.Stores.YesSqlStore`2.ToQuery(ISession session, ISpecification`1 specification)
   at Elsa.Persistence.YesSql.Stores.YesSqlStore`2.Query(ISession session, ISpecification`1 specification, IOrderBy`1 orderBy, IPaging paging, CancellationToken cancellationToken)
   at Elsa.Persistence.YesSql.Stores.YesSqlStore`2.FindManyAsync(ISpecification`1 specification, IOrderBy`1 orderBy, IPaging paging, CancellationToken cancellationToken)
   at Elsa.Persistence.YesSql.Stores.YesSqlStore`2.FindManyAsync(ISpecification`1 specification, IOrderBy`1 orderBy, IPaging paging, CancellationToken cancellationToken)
   at Elsa.Activities.Http.Middleware.HttpEndpointMiddleware.InvokeAsync(HttpContext httpContext, IWorkflowLaunchpad workflowLaunchpad, IWorkflowInstanceStore workflowInstanceStore, IWorkflowRegistry workflowRegistry, IWorkflowBlueprintReflector workflowBlueprintReflector, IEnumerable`1 contentParsers)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)

```

**Comment by sfmskywalker** [comment]
Fixed with 46259b30141a7047a673c9909255e89c16239ee5

**Comment by matt4446** [comment]
Thanks for picking that up so quickly
